test/workflow.vader

```diff
@@ -1055,15 +1059,13 @@ Execute (Post-update hook output; success and failure):
   AssertEqual 'x Post-update hook for vim-easy-align ... Exit status: 127', getline(6)
   q
 
-Execute (Post-update hook output; invalid type or funcref):
+Execute (Post-update hook output; invalid funcref):
   call plug#begin()
-  Plug 'junegunn/vim-easy-align', { 'do': 1 }
   Plug 'junegunn/vim-pseudocl',   { 'do': function('call') }
   call plug#end()
 
   silent PlugInstall! 1
   AssertEqual 'x Post-update hook for vim-pseudocl ... Vim(call):E119: Not enough arguments for function: call', getline(5)
-  AssertEqual 'x Post-update hook for vim-easy-align ... Invalid hook type', getline(6)
```

Comment:
`Invalid hook type` can happen if the user updates `g:plugs` after `plug#end` and before `PlugInstall`.